### PR TITLE
MOL-466

### PR DESCRIPTION
**Pull request by elmgren**

### CODE REVIEW

File: mottak-arkiv-service/app/database/repositories/arkivkopi_repository.py [hunk 1]
```diff
@@ -21,6 +21,21 @@ def get_by_id(db: Session, id_: int) -> Arkivkopi_DBO:
...
+    For every arkivuttrekk uploaded there is one invitasjon.
+    For each invitasjon that is used to upload an archive there is one overforingspakke.
+
+    There can be multiple copies made of an overforingspakke and you can choose if you want to make a copy of
+    the unpacked archive (bucket) or the tarfile itself. To separate these types of copies you use the
+    boolean parameter "is_object".
+
+    To identify the current arkivkopi of an overforingspakke we use:
+        - the corresponding "invitasjon_id"
+        - the boolean paramater "is_object" to choose between copy types.
```
Comment (sse-computas) [inline mottak-arkiv-service/app/database/repositories/arkivkopi_repository.py:36]: 'paramater' -> 'parameter'

File: mottak-arkiv-service/app/routers/overforingspakke.py [hunk 2]
```diff
@@ -16,3 +21,75 @@
...
+    except SASTokenPreconditionFailed as err:
+        raise HTTPException(status_code=status.HTTP_412_PRECONDITION_FAILED, detail=err.message)
+    except ArkivkopiRequestFailed as err:
+        raise HTTPException(status_code=status.HTTP_500_INTERNAL_SERVER_ERROR, detail=err.message)
+
+
+@router.get('/{overforingspakke_id}/bestill_nedlasting/tar/status',
+            status_code=status.HTTP_200_OK,
+            response_model=Arkivkopi,
+            summary='Hent status for nedlasting av tar filen for et arkivuttrekk til on-prem server')
```
Comment (sse-computas) [inline mottak-arkiv-service/app/routers/overforingspakke.py:88]:
```suggestion
            summary='Hent status for nedlasting av tar-filen for et arkivuttrekk til on-prem server')
```

**Review by omBratteng — CHANGES_REQUESTED** [review]
DTO for Arkivkopi burde få ny kommentar
`app/routers/dto/Arkivkopi.py`
```python
"""
Used as the response model for GET "/arkivuttrekk/{id}/bestill_nedlasting/status"
"""
```

**Comment by elmgren** [comment]
> DTO for Arkivkopi burde få ny kommentar
> `app/routers/dto/Arkivkopi.py`
> 
> ```python
> """
> Used as the response model for GET "/arkivuttrekk/{id}/bestill_nedlasting/status"
> """
> ```

Done

**Review by omBratteng — APPROVED** [review]

**Pull request merged**